src/sagemaker_inference/torchserve.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This package is designed to be framework agnostic.
Torchserve belongs either in pytorch-serving or in its own repository.

Reply:
Closing this PR as this has been moved to https://github.com/aws/sagemaker-pytorch-inference-toolkit/pull/79